libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -263,6 +263,7 @@ singleton AP_Vehicle method set_target_angle_and_climbrate boolean float -180 18
 singleton AP_Vehicle method get_circle_radius boolean float'Null
 singleton AP_Vehicle method set_circle_rate boolean float'skip_check
 singleton AP_Vehicle method set_steering_and_throttle boolean float -1 1 float -1 1
+singleton AP_Vehicle method get_steering_and_throttle boolean float'Null float'Null
```

Comment:
If you don't actually need the bool.
```suggestion
singleton AP_Vehicle method get_steering_and_throttle void float'Ref float'Ref
```

Reply:
@IamPete1, ok, thanks.  I was thinking that I'd need the boolean because it could be called while using non-Rover vehicles that don't implement the method.